packages/utils/add-script.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from "path";
+import * as fs from "fs";
+
+/**
+ * Utility function to add a script in a package.json file given a key and a value
+ *
+ * @param {String} key - key of the script to be added in package.json
+ * @param {String} value - command to be executed on running that script
+ * @returns {void}
+ */
+
+export default function addScript(key: string, value: string): void {
+  const localDir = process.cwd();
+  // eslint-disable-next-line
```

Comment:
ESLint suggests to use import in the next line but it doesn't work for some reason.

Reply:
Because the `require` it's used, usually, to resolve modules. Here you want to read a file, so you should use `fs.readFile` instead

Reply:
😮 ofcourseeee